core/src/main/scala/org/apache/spark/executor/CoarseGrainedExecutorBackend.scala

```diff
@@ -259,12 +260,13 @@ private[spark] class CoarseGrainedExecutorBackend(
   }
 
   private def decommissionSelf(): Boolean = {
-    logInfo("Decommissioning self w/sync")
+    val msg = "Decommissioning self w/sync"
```

Comment:
I know it's from original commit, but shall we use `with` instead of `w/`? Log info is exposed to end-users. I'm afraid some of them may not understand it.